src/customTheme.ts

```diff
@@ -101,36 +103,81 @@ const tertiaryTextDarkmode = coolGray['400']
 // ***** Collect constants, put them in a customTheme *****
 
 const themeConfig: ThemeConfig = {
-  initialColorMode: 'light',
   useSystemColorMode: !!process.env.NEXT_PUBLIC_FEATURE_FLAG_COLOR_MODE,
 }
 
-const customTheme = extendTheme({
-  colors: {
-    blue: blueColor,
-    primary: primaryColor,
-    secondary: blueColor,
-    coolGray,
-    fuchsia,
-    sky,
-    primaryTextLightmode,
-    secondaryTextLightmode,
-    tertiaryTextLightmode,
-    primaryTextDarkmode,
-    secondaryTextDarkmode,
-    tertiaryTextDarkmode,
-  },
-  fonts: {
-    heading: '"Inter", sans-serif;',
-    body: '"Inter", sans-serif;',
-    mono: '"Roboto Mono", monospace;',
-  },
-  styles: {
-    global: {
-      'html, body, #__next': { height: '100%' },
+const customTheme = extendTheme(
+  {
+    colors: {
+      blue: blueColor,
+      primary: primaryColor,
+      secondary: blueColor,
+      coolGray,
+      fuchsia,
+      sky,
+      primaryTextLightmode,
+      secondaryTextLightmode,
+      tertiaryTextLightmode,
+      primaryTextDarkmode,
+      secondaryTextDarkmode,
+      tertiaryTextDarkmode,
+    },
+    fonts: {
+      heading: '"Inter", sans-serif;',
+      body: '"Inter", sans-serif;',
+      mono: '"Roboto Mono", monospace;',
+    },
+    styles: {
+      global: {
+        'html, body, #__next': { height: '100%' },
+      },
+    },
+    config: themeConfig,
+    components: {
+      Link: {
+        baseStyle: (props: Dict) => {
+          return { color: mode('primary.500', 'primary.200')(props) }
+        },
+      },
+      Button: {
+        variants: {
+          cta: (props: Dict) => {
+            return {
+              fontWeight: 'black',
+              fontSize: '2xl',
+              letterSpacing: 'tight',
+              p: 6,
+              size: 'lg',
+              boxShadow: '0px 4px 4px rgba(0, 0, 0, 0.25)',
+              borderRadius: '2xl',
+              bg: mode('primary.200', 'primary.900')(props),
+              color: mode('primary.900', 'white')(props),
+              _hover: {
+                bg: mode('primary.100', 'primary.700')(props),
+                cursor: 'pointer',
+              },
+              _active: {
+                bg: mode('primary.200', 'primary.800')(props),
+                boxShadow: '0px 2px 2px rgba(0, 0, 0, 0.25) !important',
+              },
```

Comment:
I think there's a problem with the colors for the CTA button now.

What we had before:

![original-colors](https://user-images.githubusercontent.com/34918129/148224913-9b1ff4e0-86b8-4ae2-b9b9-ff71df7243a4.png)

What we have in this PR's deployment preview (note the black color for the text in the button):

![black-color-on-rework](https://user-images.githubusercontent.com/34918129/148224938-5b4be6f2-4f4f-41f9-96af-35433fc614da.png)

Reply:
Good catch! I should have broken this in one of my latest changes, let me investigate it.

Reply:
Fixed! I was using the "primary" token but that still references to the old primary color, so I'm using "fuchsia" for now.